pkg/virt-controller/watch/vmi_test.go

```diff
@@ -847,7 +845,16 @@ var _ = Describe("VirtualMachineInstance watcher", func() {
 			addActivePods(vmi, pod.UID, "")
 
 			controller.Execute()
-			expectPodAnnotations(pod, HaveKeyWithValue(deviceinfo.NetworkPCIMapAnnot, `{"`+sriovNetworkName+`":"`+selectedPCIAddress+`"}`))
+
+			Expect(pod.Annotations).To(HaveKey(downwardapi.NetworkInfoAnnot))
```

Comment:
I would just check for the existence of the key, the rest is covered in the pod_annotations_test.